.github/CODEOWNERS

```
@@ -1 +1,4 @@
-* @confluentinc/cli
+*                   @confluentinc/cli
+pkg/flink/ @confluentinc/cloud-surfaces
+pkg/ccloudv2/flink.go @confluentinc/cli @confluentinc/cloud-surfaces
+pkg/ccloudv2/flink_gateway.go @confluentinc/cli @confluentinc/cloud-surfaces
```

I'd recommend removing the bottom two. You're actually adding *more* friction there since we're going from only needing CLI review, to needing *both* CLI and Flink review.
```suggestion
*          @confluentinc/cli
pkg/flink/ @confluentinc/cloud-surfaces
```

---

Ops, right! Created it in a hurry. What do you think of

```
pkg/ccloudv2/flink.go @confluentinc/cli 
pkg/ccloudv2/flink_gateway.go @confluentinc/cloud-surfaces
```

We don't really interact with flink.go but we usualy the ones making changes to flink_gateway and might do some refactor there to use some new features for sorting results and so on.

If you don't feel comfortable, I'd do as you mentioned and let you as onwers for both :)

---

Sure! Although the top one is redundant since it's handled by line 1

---

Updated